blocks/generator.go

```diff
@@ -60,6 +63,29 @@ func defaultConfig() Config {
 	}
 }
 
+type stats struct {
+	lock       sync.Mutex
+	blockCount map[types.EpochID]int
+}
+
+func (s *stats) Add(block *types.Block) {
+	s.lock.Lock()
+	defer s.lock.Unlock()
+	epoch := block.LayerIndex.GetEpoch()
+	s.blockCount[epoch]++
+	delete(s.blockCount, epoch-2)
+}
```

Comment:
i think this metric might be useful for this component, but in general it has some limitations. node can download and apply block bypassing this generator and it should be good enough for monitoring purposes

Reply:
i've updated the metrics to track block certificate instead. after all that's what we are looking for.